pcfuncs/animation/stamps/brand.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image, ImageDraw, ImageFont
```

Comment:
Perhaps we should move `stamps` to `funclib` instead of `animation`. I think this would be good to include in the `image` endpoint as well, and a follow up PR could use the same logic.

Reply:
Done in f917e5f